ci-operator/step-registry/hypershift/eval-agents/run/hypershift-eval-agents-run-commands.sh

```
@@ -0,0 +1,18 @@
+
+cd /go/src/github.com/openshift/hypershift
+
+MAKE_ARGS="EVAL_OUTPUT=${ARTIFACT_DIR}/results.xml"
+
+if [ -n "${EVAL_FOCUS:-}" ]; then
+    echo "Filter: ${EVAL_FOCUS}"
+    MAKE_ARGS="${MAKE_ARGS} EVAL_FILTER=${EVAL_FOCUS}"
+fi
+
+echo "Running eval-agents with promptfoo..."
+make eval-agents ${MAKE_ARGS}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)